d3ploy/timeseries_inst.py

```diff
@@ -259,10 +266,12 @@ def predict_demand(self, commod, time):
                 demand = CALC_METHODS[self.calc_method](self.commodity_demand[commod],
                                                         back_steps=self.back_steps,
                                                         degree=self.degree)
+            elif self.calc_method in ['sw_seasonal']:
```

Comment:
Can we maybe just include this in the DO calc methods and have a dummy parameter `back_steps` that doesn't do anything?